src/pretalx/orga/templates/orga/submission/speakers.html

```diff
@@ -60,6 +61,15 @@ <h5>{% translate "Other proposals by this speaker:" %}</h5>
                 {% endif %}
             </p>
             {% if can_edit_speakers %}
+                {% if can_mark_speaker and submission.state in 'accepted,confirmed' %}
```

Comment:
Would probably make sense to extract this block into a template included in the three templates.